arch/simd_vector.cc

```diff
@@ -37,7 +37,9 @@ void vec_scale(const float * x, float k, float * r, size_t n)
 {
     size_t i = 0;
 
-    if (false) ;
+        if (false)
+        ;
```

Comment:
It avoids syntax errors when you have a long if with different parts enabled by the preprocessor.  It means that you have all `else if` branches so it doesn't matter which ones are disabled.  Otherwise you get a syntax error if you disable the first.